packages/agent-toolkit/src/lib/constants.ts

```diff
@@ -0,0 +1,5 @@
+import type { ToolkitContext } from './types';
+
+export const defaultToolkitContext: ToolkitContext = {
+  allowPrivateMetadata: false,
```

Comment:
JSDoc comments would be much appreciated :)

Reply:
done!